requirements.txt

```diff
@@ -18,6 +18,7 @@ django-rest-swagger==0.3.3
 django-summernote==0.6.8
 django-test-without-migrations==0.4
 djangorestframework==3.1.3
+factory-boy==2.11.1
```

Comment:
We usually create a `requirements-dev.txt` with the following content:

```
-r requirements.txt

factory-boy==2.11.1
ipython==4.0.0
ipython-genutils==0.1.0
```

This is intended for dev use and will install all the traditional requirements from `requirements.txt` plus all the dev packages defined below that. In this way we don't have to install dev requirements in production and we can have smaller Docker images (in the future).